### DISCUSSION TITLE
Changing AppBar orientation

### DISCUSSION THREAD

**Discussion by erickmiyazaki** [microsoft/fluentui-blazor#2750] [Q&A] [answered]
Hi,

Is there any way I can change the AppBar orientation to horizontal?

**Comment by vnbaaij** [comment, marked as answer]
No, we have not designed nor built it to be used like that.

**Reply by erickmiyazaki** [reply]
Thank you.
I will suggest this as a new feature, maybe there is enough requests to get this implemented.
